README.md

```diff
@@ -342,6 +342,30 @@ While several Ruby books suggest the first style, the second is much more promin
 
 Refactoring is even better. It's worth looking hard at any code that explicitly checks types.
 
+- For single-line lambdas, `lambda` or `->` may be used.
+
+```ruby
+  scope :example, -> { where(:a => b) }
+
+  # Or
+
+  scope :example, lambda { where(:a => b) }
+```
+
+- For multi-line lambdas, prefer `lambda do ... end`
+
+```ruby
+  # bad
+  scope :long_example, lambda {
+    # ...
+  }
+
+  # good
+  scope :long_example, (lambda do
```

Comment:
Unless there's a strong argument against this, I'd say we stick to `{ }` for all `lambda` declarations. I'm actually not sure I've ever seen `lambda do … end` in the wild